frontend/src/api/collimator/models/users/existing-user.ts

```diff
@@ -4,17 +4,20 @@ import { ClassProperties } from "../class-properties";
 export class ExistingUser {
   readonly id: number;
   readonly email: string;
+  readonly oidcSub: string | null;
```

Comment:
Given we deployed development versions (including migrations) it will be easier  to address this in a subsequent PR, https://classroomrt.atlassian.net/browse/CRT-63.